inc/ADC.h

```diff
@@ -0,0 +1,44 @@
+#ifndef ADC_H_
+#define ADC_H_
+
+extern uint16_t ADC_Values[17][5];
+extern uint8_t Read_Position;
+extern uint8_t ADC_count;
+double average;
+
+typedef enum ADC_sensors{
+	//current
+		Curr_ADC1 = 1,
```

Comment:
If this starts at zero, you can get rid of your switch statement below. See comment below for more details. Let me know if setting it to 0 will break things elsewhere in the code.